images_config/ks-robot-slave.cfg

```diff
@@ -248,6 +218,9 @@ fi
 
 ################ NETWORK SETUP ################ 
 %post --interpreter=/bin/bash --logfile /root/ks-post.log
+
+apt-get purge modemmanager -y
```

Comment:
this line has nothing to do with the network setup

Reply:
split modemmanager into standalone section before apt-cacher in https://github.com/ipa320/setup_cob4/pull/158/commits/3190d5f89bffba7b53b1e023e44eef51291e1093